doc/source/fmpz.rst

```diff
@@ -596,14 +592,8 @@ Comparison
 
 .. function:: int fmpz_cmp(const fmpz_t f, const fmpz_t g)
 
-    Returns a negative value if `f < g`, positive value if `g < f`, 
-    otherwise returns `0`.
-
 .. function:: int fmpz_cmp_ui(const fmpz_t f, ulong g)
 
-    Returns a negative value if `f < g`, positive value if `g < f`, 
-    otherwise returns `0`.
-
 .. function:: int fmpz_cmp_si(const fmpz_t f, slong g)
 
     Returns a negative value if `f < g`, positive value if `g < f`, 
```

Comment:
The other docstring is the better one as it says what happens otherwise.

Reply:
I haven't changed the docstring for this function, and I don't see a difference between this docstring and the other ones for `fmpz_cmp` and `fmpz_cmp_ui`. Or did you mean some other docstring perhaps?

Reply:
Oh I see the diff cuts off there. My mistake.